### PR TITLE
fix linkedin regexes

### LINKED ISSUE #17: Fix linkedin recruiter and add /talent url regex
There is a bug in the linkedin recruiter URL regex, it lacks the "profile" part

This is how the link should be
https://www.linkedin.com/recruiter/profile/476162262,HHNH,name

Also, the URL regex for the "talent" endpoint is missing
https://www.linkedin.com/talent/profile/AEEAABxhqNYBJa6QzJWImzsC_q6ugSZg2H6s7pA

Please let me know if you'd like a pull request or you'll fix them by yourself :)

### PR DESCRIPTION
**Pull request by Ronserruya**
Closes #17 

Tested this on a list of a few thousands of user links, hope this helps :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,15 +21,27 @@ Regex for pages like someuser.github.io.
 Use https://api.github.com/users/{user_login} (60 requests/hour unauthenticated)
 
 ## Linkedin
-    http(s)?:\/\/([\w]+\.)?linkedin\.com\/in\/[A-z0-9_-]+\/?
+
+    http(s)?:\/\/([\w]+\.)?linkedin\.com\/in\/[\w\-À-ÿ%]+(\/.*)?
 RegEx for public URLs.
 
-    http(s)?:\/\/([\w]+\.)?linkedin\.com\/pub\/[A-z0-9_-]+(\/[A-z0-9]+){3}\/?
+    http(s)?:\/\/([\w]+\.)?linkedin\.com\/pub\/[\w\-À-ÿ%]+(\/\w+){3}(\/.*)?
```
Comment (lorey) [inline README.md:28]: Hey @Ronserruya, thanks for the PR. Could you help me understand the `À-ÿ` part?

Comment (Ronserruya) [reply README.md:28]: Sure, \w gets all English letters, but you need À-ÿ to catch accented letters, that's for people with these letters in their name, for example 

www.linkedin.com/in/mark-simon-müller-81a83743

**Comment by lorey** [comment]
Hey @Ronserruya, sorry I changed the structure but it seemed right to structure the data and add tests. Could you please add your regexes to the regexes.json file or just post some (anoymized) sample URLs here, so I can add and verify them myself?

Tried to find examples with google but couldn't find any, sorry!

**Comment by lorey** [comment]
Closing this for now, feel free to create a new issue or PR! :100:

**Comment by Ronserruya** [comment]
No problem, I'll get to it in the next few days :)

**Pull request closed without merging**